umm-spec-lib/test/cmr/umm_spec/test/umm_g/generators.clj

```diff
@@ -26,11 +26,20 @@
                  coords-gen
                  coords-gen))))
 
+(def umm-g-data-granules
```

Comment:
We should use the existing generator in umm-lib. This is not creating the correct umm-lib granule model DataGranule record.

Reply:
The effort to create this generator is pretty much the same as adding in the umm-lib to UMM-G DayNightFlag mapping. So we should tread this generator with the DayNightFlag mapping. :)

Reply:
Its really a guess as to what we should do, I erred on the side of keeping the current schema.